.github/workflows/main.yml

```diff
@@ -62,20 +62,62 @@ jobs:
   integration-tests-flextesa:
     runs-on: ubuntu-latest
     continue-on-error: true
+    strategy:
+      matrix:
+        include:
+          - protocol: Kathmandu
+            testnet: kathmandunet
+            testnet_uppercase: KATHMANDUNET
+          - protocol: Lima
+            testnet: limanet
+            testnet_uppercase: LIMANET
+    env:
+      flextesa_docker_image: oxheadalpha/flextesa:latest
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
         node-version: 16
-    - run: docker run --rm --name my-sandbox --detach -p 20000:20000 -e block_time=1 oxheadalpha/flextesa:latest kathmandubox start
+
+    ## The 4 Bootstrap Accounts (alias, pk, pkh, sk)
+    # alice,edpkvGfYw3LyB1UcCahKQk4rF2tvbMUk8GFiTuMjL75uGXrpvKXhjn,tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb,unencrypted:edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq
+    # bob,edpkurPsQ8eUApnLUJ9ZPDvu98E8VNj4KtJa1aZr16Cr5ow5VHKnz4,tz1aSkwEot3L2kmUvcoxzjMomb9mvBNuzFK6,unencrypted:edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt
+    # charlie,edpkuvMuRuZ6ZbAquJH1XxBFfUmuBFz1zp9ENEqjCVgLp3NcY3Ww9M,tz1RDVcKmFcqzvTpJirvg4JaUVgZbjtnRT26,unencrypted:edsk3RgWvbKKA1atEUcaGwivge7QtckHkTL9nQJUXQKY5r8WKp4pF4
+    # donald,edpkvXGp1BMZxHkwg3mKnWfJYS6HTJ5JtufD8YXxLtH8UKqLZkZVun,tz1eSWp4B9s1qhtNMMNXAkaf2oqCnDHd2iAm,unencrypted:edsk3S8mG2sSBmSRbikAcZVLCz4SrCq4DjmsQRic6MGktqNFijfrS2
+    - name: Generate Flextesa bootstrap accounts
+      run: |-
+        echo "alice=$(docker run --rm ${flextesa_docker_image} flextesa key alice)" >> $GITHUB_ENV 
+        echo "bob=$(docker run --rm ${flextesa_docker_image} flextesa key bob)" >> $GITHUB_ENV
+        echo "charlie=$(docker run --rm ${flextesa_docker_image} flextesa key charlie)" >> $GITHUB_ENV
+        echo "donald=$(docker run --rm ${flextesa_docker_image} flextesa key donald)" >> $GITHUB_ENV
+    - name: Provision Flextesa ${{ matrix.protocol }} container
+      run: |-
+        docker run \
+        --rm \
+        --name my-sandbox \
+        --detach \
+        -p 20000:20000 \
+        ${flextesa_docker_image} \
+        flextesa mini-net \
+          --root /tmp/mini-box --size 1 \
+          --set-history-mode N000:archive \
+          --number-of-b 1 \
+          --balance-of-bootstrap-accounts tez:100_000_000 \
+          --time-b 1 \
+          --add-bootstrap-account="${alice}@2_000_000_000_000" \
```

Comment:
An should be able to leverage these accounts for drain_delegate testing right?

Reply:
Correct. We can provision additional bootstrap accounts with these flags